envs/example/defaults.yml

```diff
@@ -219,7 +219,7 @@ glance:
   api_workers: 1
   registry_workers: 1
   logging:
-    debug: True
+    debug: False
```

Comment:
So, for the examples stuff, we leave debug on to help with development. The examples config are not used by production installs, those have their own config in a private repository. The role default is where it should be set, as that's what we say is the way it "should" be, so deployers have to make an explicit change to deviate.